client/extensions/zoninator/components/settings/zone/index.jsx

```diff
@@ -60,7 +77,11 @@ class Zone extends Component {
 	saveZoneFeed = ( form, data ) =>
 		this.props.saveFeed( this.props.siteId, this.props.zoneId, form, data.posts );
 
-	renderContent() {
+	locked = () =>
```

Comment:
Maybe I'm misunderstanding, but this logic looks backwards. Shouldn't `locked` be true if `lockBlocked` is _true_, or if `lockExpires` is _less than_ the current time?

I may be wrong, but I think either the logic needs to be reversed here, or the method should be called `unlocked` instead of `locked`.

Reply:
Hmm, let's put it like that:  
`locked` is the desired state of a zone to be edited. A zone is `locked` by when:
- It isn't `blocked` by another user
- The lock hasn't timed out

Hopefully it makes more sense now. I'm open to other name suggestions but I think changing it to `unlocked` would just mean switching the operators, not necessarily improving the clarity.

Reply:
Ahh I understand. It's a bit ambiguous. When I see "locked", I assume it means "locked by someone else" (and thus not accessable), whereas in this case it means "locked by me".

I wonder if we could update the name to make that more clear? Just saying it is `locked` could cause confusion.

Reply:
How about renaming the function to `disabled` & moving the negation into the function ? Apparently it's actually the only thing I used it for 😄

Reply:
That sounds perfect!

Reply:
Done 🙂